src/httpclient.c

```diff
@@ -715,7 +715,7 @@ int main(int argc, char **argv)
 #endif
     }
 
-    if (ctx.http3 != NULL) {
+    if (ctx.protocol_selector.ratio.http3 > 0) {
```

Comment:
Should this if block better be changed to `#if H2O_USE_LIBUV \n #else ... #endif`? The reason I wonder is because that's the way the QUIC context is being initilaized at the beginning of this function.

Though I would not expect practical difference between the two approaches, because ATM the only case where we create QUIC connections is when the protocol selector is set to non-zero, and because when it is set to non-zero `h2o_quiy_close_all_connections` becomes a no-op.

Reply:
Done in c6565f8

Reply:
Thank you for the changes! I like the fact that the number of ifs have been reduced.